library/src/main/kotlin/it/krzeminski/githubactions/actions/madhead/CheckGradleVersion.kt

```diff
@@ -0,0 +1,15 @@
+package it.krzeminski.githubactions.actions.madhead
+
+import it.krzeminski.githubactions.actions.Action
+
+data class CheckGradleVersion(
```

Comment:
Could we put `V1` explicitly in the class name, like in case of e.g. [`CheckoutV2`](https://github.com/krzema12/github-actions-kotlin-dsl/blob/main/library/src/main/kotlin/it/krzeminski/githubactions/actions/actions/CheckoutV2.kt)?
```suggestion
data class CheckGradleVersionV1(
```